crates/core/tedge/src/cli/reconnect/command.rs

```diff
@@ -55,7 +53,10 @@ impl From<&ReconnectBridgeCommand> for ConnectCommand {
     fn from(reconnect_cmd: &ReconnectBridgeCommand) -> Self {
         ConnectCommand {
             config_location: reconnect_cmd.config_location.clone(),
-            config_repository: reconnect_cmd.config_repository.clone(),
+            // FIXME: cloning the config leads to the following error:
+            //        expected `TEdgeConfig`, found `TEdgeConfigReader`
+            // config: reconnect_cmd.config.clone(),
+            config: TEdgeConfig::new(reconnect_cmd.config_location.clone()).unwrap(),
```

Comment:
@jarhodes-sag any ideas on why this error and how this can be fixed?

Reply:
Fixed by https://github.com/thin-edge/thin-edge.io/pull/2735/commits/e5eec7519f5315fd0a4123236834bd7ddcacbdb3